src/libPMacc/include/algorithms/math/floatMath/trigo.tpp

```diff
@@ -58,6 +58,21 @@ struct Cos<float>
 };
 
 template<>
+struct ACos<float>
+{
+    typedef float result;
+
+    HDINLINE float operator( )(const float& value )
```

Comment:
Space is missing after the parentheses. `HDINLINE float operator( )( const....)`